MinimedKit/PumpModel.swift

```diff
@@ -47,6 +47,10 @@ public enum PumpModel: String {
         return generation >= 23
     }
     
+    public var hasMySentry: Bool {
```

Comment:
there might be a few places in code (RileyLinkBLEKit? Loop? I don't remember) that can be updated to take advantage of this instead of using `larger` or something else as a proxy...